compiler/rustc_middle/src/query/mod.rs

```diff
@@ -197,15 +197,15 @@ rustc_queries! {
 
     /// Gives access to the HIR node's parent for the HIR owner `key`.
     ///
-    /// This can be conveniently accessed by methods on `tcx.hir()`.
```

Comment:
Note for future work: Now that there are lots of useful `.hir_*` methods on tcx, having these queries in the same “namespace” feels like a bit of a footgun.